ArdentHQ/dashbrd
# refactor: nft details page

# [[nfts] Move NFT actions on the right side, remove items from image](https://app.clickup.com/t/861nam5mr)

## Summary

- NFT Details page has been refactored to match [our new designs in Figma file](https://www.figma.com/file/2Rr7zzoeGGiHYsROvfHaGe/Dashboard?type=design&node-id=21873-75966&mode=design&t=vcoRB84zBaBlwbiQ-0)

> **_NOTE:_**  Marketplace buttons are not in this ticket, they correspond to [[nfts] Add marketplace links to NFTs](https://app.clickup.com/t/861nam1a8)


## What changes are being made?

- `ParseIcon` can now receive `2xs` size for button icons.
- New components:
   - `NftOwner` to display the wallet that owns an NFT and the correct link to its own block explorer.
   - `NftActions` to contain the new action buttons, including the new `View NFT on` icon to redirect to the block explorer.
   - `NftBasicInfo` to contain the collection and NFT basic info.
- `NftImage` and `GalleryCard` components have been refactored to support these changes.
- When clicking an NFT image, it will open in a new tab.
- When hovering an NFT image, it will display the magnifying glass cursor.
- Actions are not on NFT image anymore. 
- Styles for various NFT view related components have been changed.
- New translations for each with copy changes for each new component.
- New unit tests for all these changes.

## How to reproduce

1. Run the app in `local` or `test_e2e` mode. Make sure you have a wallet with multiple collections and NFTs.
2. Go to `Collections` page.
3. Select any collection.
4. Select any NFT and see the magic ✨ 

## Views

- 320px:

<img width="737" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/cd1dfd86-bb69-410a-8b1d-44d24a8ab3c8">

- 640px:

<img width="726" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/92d6608f-a377-4dbc-aeea-b1c3be690eff">

- 768px:

<img width="655" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/2e25d200-5229-4546-a2a2-c1057b8c6920">

- 960px:

<img width="629" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/e094e2d5-2919-406f-85f7-8a0bef870baa">

- 1024px:

<img width="670" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/ffb51b5d-8bae-4a35-aaef-9d3465048e1e">

- 1280px:

<img width="1279" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/84feec86-ea86-4071-80e8-73b23f36b17e">

- 1440px and more:

<img width="1111" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/55117912/4f772400-7075-4e31-add2-e72875677bbc">


## Checklist
